include/nogdb_types.h

```diff
@@ -197,12 +200,15 @@ namespace nogdb {
 
     class Record {
     public:
+
+        using RecordPropertyType = std::map<std::string, Bytes>;
```

Comment:
Why in `class Record`? 

For `RecordPropertyType`, I think it made me a little bit confused at the first time I saw. `RecordProperty` may be a better name (when it is placed outside `class Record` so the developers can use it as `nogdb::RecordProperty`).

Reply:
Change the name to `DataType` for less confusion.